kitsune/upload/tests/test_views.py

```diff
@@ -117,73 +116,73 @@ def test_delete_image_owner(self):
         assert not ImageAttachment.objects.exists()
 
     def test_delete_no_image(self):
         """Trying to delete a non-existent image 404s."""
         r = self._make_post_request(args=[123])
         eq_(404, r.status_code)
         data = json.loads(r.content)
-        eq_('error', data['status'])
+        eq_("error", data["status"])
 
     def test_invalid_image(self):
         """Make sure invalid files are not accepted as images."""
-        with open('kitsune/upload/__init__.py', 'rb') as f:
+        with open("kitsune/upload/__init__.py", "rb") as f:
             r = self._make_post_request(image=f)
 
         eq_(400, r.status_code)
         json_r = json.loads(r.content)
-        eq_('error', json_r['status'])
-        eq_('Invalid or no image received.', json_r['message'])
-        eq_('The submitted file is empty.', json_r['errors']['image'][0])
+        eq_("error", json_r["status"])
+        eq_("Invalid or no image received.", json_r["message"])
+        eq_("The submitted file is empty.", json_r["errors"]["image"][0])
 
     def test_invalid_image_extensions(self):
         """
         Make sure invalid extensions aren't accepted as images.
         """
-        with open('kitsune/upload/tests/media/test_invalid.ext', 'rb') as f:
+        with open("kitsune/upload/tests/media/test_invalid.ext", "rb") as f:
             r = self._make_post_request(image=f)
 
         eq_(400, r.status_code)
         json_r = json.loads(r.content)
-        eq_('error', json_r['status'])
-        eq_('Invalid or no image received.', json_r['message'])
+        eq_("error", json_r["status"])
+        eq_("Invalid or no image received.", json_r["message"])
         assert (
-            json_r['errors']['image'][0] ==
-            "File extension 'ext' is not allowed. Allowed extensions are: 'jpg, jpeg, png, gif'.")
+            json_r["errors"]["image"][0]
+            == "File extension 'ext' is not allowed. Allowed extensions are: 'jpg, jpeg, png, gif'."  # noqa
+        )
 
     def test_unsupported_image_extensions(self):
         """
         Make sure valid image types that are not whitelisted aren't accepted as images.
         """
-        with open('kitsune/upload/tests/media/test.tiff', 'rb') as f:
+        with open("kitsune/upload/tests/media/test.tiff", "rb") as f:
             r = self._make_post_request(image=f)
 
         eq_(400, r.status_code)
         json_r = json.loads(r.content)
-        eq_('error', json_r['status'])
-        eq_('Invalid or no image received.', json_r['message'])
+        eq_("error", json_r["status"])
+        eq_("Invalid or no image received.", json_r["message"])
         assert (
-            json_r['errors']['image'][0] ==
-            "File extension 'tiff' is not allowed. Allowed extensions are: 'jpg, jpeg, png, gif'.")
+            json_r["errors"]["image"][0]
+            == "File extension 'tiff' is not allowed. Allowed extensions are: 'jpg, jpeg, png, gif'."  # noqa
```

Comment:
we can remove noqa from here

Reply:
I don't think we can, the line is longer than 99 chars without it